packages/svelte-check/src/writers.ts

```diff
@@ -159,10 +163,23 @@ export class MachineFriendlyWriter implements Writer {
                     : null;
 
             if (type) {
-                const { line, character } = range.start;
+                const { line: startLine, character: startCharacter } = range.start;
+                const { line: endLine, character: endCharacter } = range.end;
                 const fn = JSON.stringify(filename);
                 const msg = JSON.stringify(message);
-                this.log(`${type} ${fn} ${line + 1}:${character + 1} ${msg}`);
+                if (this.isVerbose) {
+                    const stringifiedCodeDescription = JSON.stringify(codeDescription);
+                    const stringifiedSource = JSON.stringify(source);
+                    this.log(
+                        `${type} ${fn} ${startLine + 1}:${startCharacter + 1} ${endLine + 1}:${
+                            endCharacter + 1
+                        } ${msg} "${code ?? ''}" ${stringifiedCodeDescription ?? '""'} ${
+                            stringifiedSource ?? '""'
+                        }`
```

Comment:
Something like the ndjson format would be easier for a machine to parse reliably compared to an arbitrary space separated line.

Reply:
I'm totally ok with it, I was sticking to that to keep it similar to the `machine` version...I can change that

Reply:
Obviously let me know if that's what we want

Reply:
Wait for a maintainer to confirm, but a standardized format would be my recommendation. I don't think you'll need a separate dependency for it, just `JSON.stringify(thing)` without any indentation should output it on a single line that complies with ndjson.